<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateItem</class>
 <widget class="QDialog" name="updateItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1469</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_loadInfo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>161</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>641</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>341</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sName_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sQuantity_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sStatus_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>DOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sDOP_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>DOS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sDOS_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>SellingPrice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sSellingPrice_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>PurchasePrice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sPurchasePrice_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
